client/gutenberg/extensions/simple-payments/editor.js

```diff
@@ -0,0 +1,36 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { registerBlockType } from '@wordpress/blocks';
+import GridiconMoney from 'gridicons/dist/money';
+
+registerBlockType( 'a8c/simple-payment', {
```

Comment:
Nit: the folder is plural `simple-payments` and block name is singular `simple-payment`. Should prolly be the same?

Looks like the product name is the plural form: https://en.support.wordpress.com/simple-payments/

Reply:
In the editor this was called _"Payment button"_ but we can easily change at any point — just the blockname and folder is more annoying to change later on.

Reply:
Done in bf153aa846ac601363ae4ea932de43bf7df6a496